### PR TITLE
Endpoints Creation

### PR DESCRIPTION
**Pull request by R0PK3**
Created Get/Post/Put/Delete endpoints with some working sample data.

### CODE REVIEW

File: API/Controllers/ElectricityController.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace trackingapi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ElectricityController : ControllerBase
+    {
+        private readonly IssueDbContext _context;       //Allow controller to access the database
```
Comment (lynxcs) [inline API/Controllers/ElectricityController.cs:13]: Comment must be /* */!!!11!! Follow coding style pls!!!!!!!!

File: API/Controllers/ElectricityController.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+        public ElectricityController(IssueDbContext context) => _context = context;
+
```
Comment (lynxcs) [inline API/Controllers/ElectricityController.cs:16]: Why so many empty lines?

File: API/Controllers/ElectricityController.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        [HttpGet("{id}")] //Get unit by it's id
+        [ProducesResponseType(typeof(Issue), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> GetById(int id)
+        {
+            var issue = await _context.Issues.FindAsync(id);
+            return issue == null ? NotFound() : Ok(issue);
+        }
+
```
Comment (Nedux) [inline API/Controllers/ElectricityController.cs:31]: Why need? From 32 line

File: API/Models/Issue.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.Models
+{
+    public class Issue
```
Comment (Afterburn7) [inline API/Models/Issue.cs:5]: What is this for?

File: API/Controllers/ElectricityController.cs (hunk 1 above)
Comment (Afterburn7) [reply API/Controllers/ElectricityController.cs:13]: Fr

Comment (lynxcs) [reply API/Controllers/ElectricityController.cs:13]: > Fr

no cap

Comment (R0PK3) [reply API/Controllers/ElectricityController.cs:13]: Sorry and thanks for the note! Already fixed.

File: API/Controllers/ElectricityController.cs (hunk 2 above)
Comment (R0PK3) [reply API/Controllers/ElectricityController.cs:16]: Empty lines were created for convenience, forgot to delete them

File: API/Controllers/ElectricityController.cs (hunk 3 above)
Comment (R0PK3) [reply API/Controllers/ElectricityController.cs:31]: Created for testing purposes, will be removed later.

File: API/Models/Issue.cs (hunk 4 above)
Comment (R0PK3) [reply API/Models/Issue.cs:5]: Placeholder, for now, tested some fancy things, will be edited later.

File: API/Controllers/ElectricityController.cs (hunk 3 above)
Comment (lynxcs) [reply API/Controllers/ElectricityController.cs]: Works for me

File: API/Controllers/ElectricityController.cs (hunk 1 above)
Comment (lynxcs) [reply API/Controllers/ElectricityController.cs]: ur welcome

**Pull request merged**